src/components/editMeasure/testCases/components/testCaseLanding/qiCore/TestCaseList.test.tsx

```diff
@@ -1836,11 +1843,11 @@ describe("TestCaseList component", () => {
     const dropZone = screen.getByTestId("file-drop-input");
     userEvent.upload(dropZone, zipFile);
 
-    await waitFor(async () => {
-      expect(importButton).toBeEnabled();
-      userEvent.click(importButton);
-      expect(setWarnings).toHaveBeenCalledWith(mockedOutcome);
-    });
+    // await waitFor(async () => {
+    //   expect(importButton).toBeEnabled();
+    //   userEvent.click(importButton);
```

Comment:
We can't confirm any response here ? Without any confirmation this test doesn't hold a value

Reply:
Changed